### PR TITLE
docs(v1.5.3): add v1.5.3 release notes to CHANGELOG

### PR DESCRIPTION
**Pull request by miparnisari**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+* Apply tags to requests that have been intentionally throttled (https://github.com/openfga/openfga/pull/1531). This will add a new field "throttled" to such logs.
+
+### Fixed
+
+* Fix panic that occurred on Check API with some authorization models and tuples (https://github.com/openfga/openfga/pull/1517)
+
+### Changed
+
+* Remove experimentals for modular models/schema 1.2 (https://github.com/openfga/openfga/pull/1520)
```
Comment (rhamzeh) [inline CHANGELOG.md:25]: Can we clarify that it is on by default?


"Modular Models (Schema 1.2) support is enabled by default and the experimental flag for it has been dropped"

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+* Bumped to Go 1.21.9 (https://github.com/openfga/openfga/pull/1523)
+
+### Security
+
+* Patch CVE-2024-31452- a critical issue where Check API returns incorrect results for some models and tuples. See the CVE report for more details.
```
Comment (jon-whit) [inline CHANGELOG.md:30]: Can you please make the CVE reference a markdown link to the published advisory?

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.5.2...v1.5.3)
+
+### Added
+
+* Apply tags to requests that have been intentionally throttled (https://github.com/openfga/openfga/pull/1531). This will add a new field "throttled" to such logs.
+
+### Fixed
+
+* Fix panic that occurred on Check API with some authorization models and tuples (https://github.com/openfga/openfga/pull/1517)
```
Comment (jon-whit) [inline CHANGELOG.md:21]: Fix stutters with the Fixed heading title. Recommend 

> Panic in Check API with some authorization models and tuples ...

**Review by jon-whit — DISMISSED** [review]

**Review by rhamzeh — DISMISSED** [review]

**Comment by rhamzeh** [comment]
Ah sorry @miparnisari , can you link to the modular models page in the docs when you say modular models?

https://openfga.dev/docs/modeling/modular-models

**Review by jon-whit — APPROVED** [review]

**Pull request merged**